numpy/lib/_function_base_impl.py

```diff
@@ -2242,7 +2242,8 @@ class vectorize:
         ``pyfunc.__doc__``.
     excluded : set, optional
         Set of strings or integers representing the positional or keyword
-        arguments for which the function will not be vectorized.  These will be
+        arguments for which the function will not be vectorized (it is recommended
+        passing both positional and keyword identifiers). These will be
```

Comment:
```suggestion
        arguments for which the function will not be vectorized. These will be
```
I tend to agree, but I'd still suggest:
- avoiding the use of parentheses in place of more appropriate punctuation and
- saving stylistic commentary for later to keep the parameter description concise.